src/api/onboard.ts

```diff
@@ -23,7 +23,7 @@ export default function onboard(storedOptions: StoredOptions, options: OnboardOp
     cobrandClientId,
     locale,
     scopes: defaultScopes,
-    redirectUri: defaultRedirectUri
+    redirectUri: defaultRedirectUri,
```

Comment:
```suggestion
    redirectUri: defaultRedirectUri
```

This file change is not necessary